internal/embedding/data.go

```diff
@@ -7,14 +7,19 @@ import (
 	"runtime"
 )
 
-// CelestiaAppV3 returns the compressed platform specific Celestia binary.
-func CelestiaAppV3() ([]byte, error) {
+// NOTE: This version must be updated at the same time as the version in the
+// Makefile.
+const CELESTIA_V3_VERSION = "v3.10.0-arabica"
+
+// CelestiaAppV3 returns the compressed platform specific Celestia binary and
+// the version.
+func CelestiaAppV3() (version string, compressedBinary []byte, err error) {
 	// Check if we actually have binary data
 	if len(v3binaryCompressed) == 0 {
```

Comment:
just thinking now, it should be possible to have a version of the multiplexer with no embedded binaries if the binaries exist in the correct directory beforehand. 

I don't think we need to change anything now, but this could be useful for testing different binaries without constantly downloading and re-building / re-embedding all the time.